CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 - Added `RepositoriesClient.GetSelfHostedRepositoryListAsync` method that will enable self hosted users to get their repository list without an access token.
+- Changed `AuthorizationType` enum values from `CloudAccessKey` and `APIServerUsernamePassword` to `CLOUD_ACCESS_KEY` and `API_SERVER_USERNAME_PASSWORD` respecitvely.
```

Comment:
This only affects the integration tests, so no need to mention in changelog

Reply:
ok removed, thanks!